packages/core/src/eventUtils.ts

```diff
@@ -48,7 +48,7 @@ export function createDoneInvokeEvent(
   output?: unknown
 ): DoneInvokeEventObject {
   return {
-    type: `done.invoke.${invokeId}`,
+    type: `xstate.done.invoke.${invokeId}`,
```

Comment:
I'd also like to propose an extra change to this. Since the same events are also generated for spawned actors, I think it would be best to use `xstate.done.actor.INVOKE_ID` here.